Utilities/File.cpp

```diff
@@ -1828,6 +1829,20 @@ bool fs::remove_all(const std::string& path, bool remove_root, bool is_no_dir_ok
 	return true;
 }
 
+std::string fs::resolve_path(std::string_view path)
+{
+	std::error_code ec{};
+	const auto result = std::filesystem::weakly_canonical(std::filesystem::u8path(path), ec);
```

Comment:
Isn't this one of previously reverted things? Probably need to test it again if it's different now.